pkg/network/protocols/http/etw_http_service.go

```diff
@@ -256,26 +251,26 @@ func init() {
 
 }
 
-func cleanupActivityIdViaConnOpen(connOpen *ConnOpen, activityId C.DDGUID) {
+func cleanupActivityIdViaConnOpen(connOpen *ConnOpen, activityId etw.DDGUID) {
 	// Clean it up related containers
 	delete(http2openConn, activityId)
 	delete(connOpen.httpPendingBackLinks, activityId)
 }
 
-func cleanupActivityIdViaConnActivityId(connActivityId C.DDGUID, activityId C.DDGUID) {
+func cleanupActivityIdViaConnActivityId(connActivityId etw.DDGUID, activityId etw.DDGUID) {
 	connOpen, connFound := connOpened[connActivityId]
 	if connFound {
 		cleanupActivityIdViaConnOpen(connOpen, activityId)
 	}
 }
 
-func getConnOpen(activityId C.DDGUID) (*ConnOpen, bool) {
+func getConnOpen(activityId etw.DDGUID) (*ConnOpen, bool) {
 	connOpen, connFound := connOpened[activityId]
 	if !connFound {
 		if captureHTTPS || HttpServiceLogVerbosity == HttpServiceLogVeryVerbose {
 			missedConnectionCount++
 			if HttpServiceLogVerbosity == HttpServiceLogVeryVerbose {
-				log.Warnf("* Warning!!!: ActivityId:%v. Failed to find connection object\n\n", formatGuid(activityId))
+				log.Warnf("* Warning!!!: ActivityId:%v. Failed to find connection object\n\n", etw.FormatGuid(activityId))
```

Comment:
We do not have DBGUID and FormatGuid anywhere at the separate utility package levels? And few other functions seems to be not ETW specific like goBytes or FormatUInt. Not super important but since you are moving/renaming you may need to consider to factor it out if we have this code elsewhere.